tesla_ble_mqtt/translations/en.yaml

```diff
@@ -0,0 +1,26 @@
+configuration:
+  mqtt_ip:
+    name: MQTT Server
+    description: MQTT Server's IP address or valid Hostname
+  mqtt_port:
+    name: MQTT Port
+    description: MQTT Listen Port
+  mqtt_user:
+    name: MQTT User
+    description: MQTT Username
+  mqtt_pwd:
+    name: MQTT Password
+    description: MQTT User's password
+  send_cmd_retry_delay:
+    name: Delay to retry a command
+    description: Delay to retry sending a command to the vehicle over BLE
+  debug:
+    name: Debug Logging
+    description: Print verbose messages to the log for debugging
+vehicle:
```

Comment:
I just noticed…. Somehow the names & descriptions below in this « vehicle » section don’t show up in the UI but in other add-ons many have a « network » section that shows names+desc just fine. I’m unclear why yet.

I propose we still merge and figure it out later, worst case we can remove the vehicle section.